### PR TITLE
[EG] Update tests

### PR DESCRIPTION
**Pull request by l0lawrence**
Updating tests to use - default set it this to az cli cred

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid/tests/test_exceptions_async.py [hunk 1]
```diff
@@ -4,31 +4,20 @@
...
-import logging
-import sys
 import os
 import json
 import pytest
-import uuid
-from datetime import datetime, timedelta
-from azure.core.exceptions import HttpResponseError, ClientAuthenticationError, ServiceRequestError
+from azure.core.exceptions import HttpResponseError, ClientAuthenticationError
 from msrest.serialization import UTC
```
Comment (kashifkhan) [inline sdk/eventgrid/azure-eventgrid/tests/test_exceptions_async.py:11]: similar to sync does this import also need to go?

Comment (l0lawrence) [reply sdk/eventgrid/azure-eventgrid/tests/test_exceptions_async.py:11]: no its used in the tests

**Comment by l0lawrence** [comment]
made a new pr for this

**Pull request closed without merging**